pwndbg/commands/dumpargs.py

```diff
@@ -38,7 +38,7 @@ def call_args() -> List[str]:
 
     for arg, value in pwndbg.arguments.get(pwndbg.aglib.disasm.one()):
         code = arg.type != "char"
-        pretty = pwndbg.chain.format(value, code=code)
+        pretty = pwndbg.chain.format(value, code=code, flags=arg.flags)
```

Comment:
Hmm... does it make sense to call `chain.format` at all if flags are there?

Maybe this should be sth like:
```py
pretty = pwndbg.chain.format(value, code=code) if not args.flags else format_flags(value, flags)
```

Reply:
I had a hard time determining whether this logic belonged in `chain.format` or elsewhere. I switched to your suggestion. I didn't see a great place to put `format_flags_argument`, so I moved it into `functions.py`.